### ISSUE TITLE
 cannot import name 'batch_norm' from 'lasagne.layers'

### ISSUE THREAD

**Issue by xiaozhangtongx** [elieJalbout/Clustering-with-Deep-learning#16]
Do you have this question? How can I solve it?

**Comment by opencvpython0805** [comment]
The website may be work. https://stackoverflow.com/questions/36088609/python-lasagne-importerror-cannot-import-batchnormlayer
